### PR TITLE
Code review for Trial Project

### PR DESCRIPTION
**Pull request by shaqdeff**
Created a tribute page using HTML and CSS.
Kindly review the code and provide feedback.

Regards.

### CODE REVIEW

**Comment by shaqdeff** [comment]
Hi @shaqdeff,
Great job so far! 👌🏽 
There is still room for improvements you can make to write better code. 

Alt Text

## Highlights
* Splendid work on the README file. ✔️💯
* Proper GitHub Flow.  ✔️
* Good commit messages.  ✔️ 

## Required Changes ♻️
* It's considered good practice to use proper HTML semantic tags. For example, you can try using  ```<section>```  or  ```<article>```  instead of  ```div```  to better structure your page. Using semantic HTML tags allows browsers to better interpret content, and improves S.E.O. 
* ID selectors are unique identifiers. Using too many  ```id=""```  selectors can potentially cause a maintenance nightmare 🥶  later on. Consider using  ```id```  selectors sparingly and only use them when necessary e.g. for unique elements you would want direct access to. You can however use the  ```class=""```  selector as many times. 



Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

***Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.***

**Review by ShadyShawkat — CHANGES_REQUESTED** [review]
## Hi @shaqdeff, 

Good job so far! 👍🏻 
There are some issues that you still need to work on to go to the next project but you are almost there!

Image

## Required Changes ♻️
- [ ] Kindly make sure to include at least 2 inline comments as a required change.


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

- [ ] [OPTIONAL]  Kindly make sure that the branch is using linters correctly and passing with green checks.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by shaqdeff** [comment]
Hi @[ShadyShawkat](https://github.com/ShadyShawkat) I humbly appreciate your review. Thank you. However, for this project, adding linter configurations and checking for linter errors was not a requirement in the project documentation. Also, it's noted that you are supposed to review my review and not the code. Kindly find attached a link to the project documentation for your corroboration.

https://github.com/microverseinc/curriculum-transversal-skills/blob/main/code-review/review_your_project.md

Thank you.

**Comment by ShadyShawkat** [comment]
Hi @shaqdeff,
Thank you for reaching out, I wasn't reviewing your code. All what I was saying is that as a reviewer, you might have to check if the branch has linters set up correctly and passing. However, it's not a part of the requirement. My intention actually was to add another required change which I forgot to add and I apologize for that.

My other required chance was:

- [ ] Kindly make sure to include at least 2 inline comments as a required change.

I apologize again for this, and so the first required change is now optional.

**Comment by shaqdeff** [comment]
Hi @ShadyShawkat, 
Thanks for the clarification. However, I'm not sure what you mean by " at least 2 inline comments". My review seems to have 6 of those. Kindly elaborate on what you mean so I can better understand how to rectify the issue.

Regards.

File: styles.css [hunk 1]
```diff
@@ -0,0 +1,65 @@
+@import url("https://fonts.googleapis.com/css2?family=Roboto&display=swap");
+* {
+  box-sizing: border-box;
+}
```
Comment (ShadyShawkat) [inline styles.css:4]: - Good job.

**Review by ShadyShawkat — COMMENTED** [review]
Hi @shaqdeff,
[This](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/code-review/articles/give_code_review_basics.md) article explains what inline comments are and how can you create them. Inline comments can be used to highlight a piece of code and add your comments on it instead of typing your required changes in your summary. (Below is an example of what an inline comment would look like)

File: index.html [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+    <title>a Tribute Page</title>
+    <link rel="stylesheet" href="css/style.css" />
+  </head>
+
+  <body>
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+
+    <main id="main">
+      <h1 id="title">Dr.Norman Borlaug</h1>
+      <p id="subtext">The man who saved a billion lives</p>
```
Comment (shaqdeff) [inline index.html:14]: Kindly use class selectors to identify elements like paragraphs and headings unless you'd like to uniquely identify something you want to have direct access to.

Comment (shaqdeff) [inline index.html:14]: Kindly use  ```class```  selectors to identify elements like paragraphs and headings unless you'd like to uniquely identify an element you want to have direct access to. Then you can use the  ```id```  selector.

**Review by shaqdeff — COMMENTED** [review]
ID selectors are unique identifiers. Using too many  ```id=""```  selectors can potentially cause a maintenance nightmare 🥶 later on. Consider using  ```id```  selectors sparingly and only use them when necessary e.g. for unique elements you would want direct access to. You can however use the  ```class=""```  selector as many times.

File: index.html (hunk 2 above)
Comment (OLIPLICHE) [reply index.html:14]: - [ ] Nice Job so far for writing one(1) inline comment👏 👏👏
- [ ] Please you need to add one (1) more inline comment to make it two (2) as was requested by the previous review.

**Review by OLIPLICHE — CHANGES_REQUESTED** [review]
Hi, @shaqdeff , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights
Great Job 👏 

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+          src="https://cdn.freecodecamp.org/testable-projects-fcc/images/tribute-page-main-image.jpg"
+          alt=""
+          id="image"
+        />
+        <p id="img-caption">
+          Dr. Norman Borlaug, third from the left, trains biologists in Mexico
+          on how to increase wheat yields - part of his life-long war on hunger.
+        </p>
+      </div>
+      <div id="tribute-info">
```
Comment (shaqdeff) [inline index.html:26]: While it's okay to use  ```div```  to contain other elements under it, please consider using more semantic HTML tags like  ```section``` ,  ```article```  and  ```aside``` .

**Comment by shaqdeff** [comment]
@OLIPLICHE Kindly review my project now and provide feedback to see if it's okay before I make a third submission.

**Review by OLIPLICHE — APPROVED** [review]
Hi @shaqdeff , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**